ouroboros-consensus-cardano/tools/db-analyser/Block/Cardano.hs

```diff
@@ -58,9 +59,9 @@ analyseBlock f =
 instance HasProtocolInfo (CardanoBlock StandardCrypto) where
   data Args (CardanoBlock StandardCrypto) =
     CardanoBlockArgs {
-        byronArgs   :: Args ByronBlock
-      , shelleyArgs :: Args (ShelleyBlock StandardShelley)
-      , alonzoArgs  :: FilePath
+        byronArgs        :: Args ByronBlock
+      , shelleyArgs      :: Args (ShelleyBlock StandardShelley)
+      , configFileAlonzo :: FilePath
```

Comment:
Hmm. On the one hand, I'd say `Args (ShelleyBlock (AlonzoEra StandardCrypto))` would fit the pattern.

But I'm also wondering if the existing pattern still makes sense, now that `CardanoBlock` is not quite as simple as it originally was ("Cardano ~ Byron + Shelley") since there are now so many sub-Shelley eras.

Reply:
Initially I want to create data instance for `Args` for Alonzo but then I realized that `ShelleyBlock` is generic for ALL shelley based blocks so that does not hold anymore. I've left if as is for now, but I'm happy to hear suggestions.

Reply:
After having conversation with @dnadales and @Jasagredo, I've added `Args` for `ShelleyBlock StandardAlonzo`. Things should now be 'consistent' with the rest of db-analyser code